domains/barshan.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "blueee04",
+        "email": "barshan2004@gmail.com"
+    },
+    "records": {
+        "CNAME": "https://blueee04.github.io/blog/"
```

CNAME doesn't accept/allow any trailing slashes (/)

```suggestion
        "CNAME": "blueee04.github.io"
```